src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/SafeHandles.cs

```diff
@@ -12,22 +13,19 @@ namespace Internal.Cryptography.Pal.Native
     /// </summary>
     internal abstract class SafePointerHandle<T> : SafeHandle where T : SafeHandle, new()
     {
-        public SafePointerHandle()
+        protected SafePointerHandle()
             : base(IntPtr.Zero, true)
         {
         }
 
-        public sealed override bool IsInvalid
+        internal static T CreateInvalid()
```

Comment:
:bulb: The introduction of `CreateInvalid()` at this location makes the diff unnecessarily complex. It would be better to move it below the `IsInvalid` property.